src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -153,6 +153,7 @@ class InterfacesUpdater(MIBUpdater):
     def __init__(self):
         super().__init__()
         self.db_conn = mibs.init_db()
+        self.multi_db_conn = mibs.init_multi_db() 
```

Comment:
>multi_db_conn [](start = 13, length = 13)

If num_asic==1, you will get `multi_db_conn=[self.db_conn]`. This is confusing.

Reply:
Fixed this, db_conn is a list of global db and all namespace DBs.
db_conn will be list with just global_db in case of single asic platform.